packages/sdk-react/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@fuels/sdk-react",
```

Comment:
I think we could call it `@fuel-wallet/react`, what do you think as a user? This package will soon include an optional `UI` for using multiple connectors once my PR is done: https://github.com/FuelLabs/fuels-portal/pull/110

Reply:
Discussed offline: we'll name it `@fuel-wallet/react` and put the package in https://github.com/FuelLabs/fuels-wallet instead.